ledger/stores/chunked_file_store.py

```diff
@@ -80,16 +93,25 @@ def getFileSize(file):
                 return chunk.numKeys
 
         path = os.path.join(dbDir, dbName)
-        os.mkdir(path)
+        if not os.path.isdir(path):
+            os.mkdir(path)
         if defaultFile:
             if self.chunkSize < getFileSize(defaultFile):
                 raise ValueError("Default file is larger than chunk size")
-            firstChunk = os.path.join(path, str(self.firstChunkIndex))
-            shutil.copy(defaultFile, firstChunk)
+            first_chunk = os.path.join(path, self._chunkIndexToFileName(self.firstChunkIndex))
+            if not os.path.exists(first_chunk.rsplit(os.path.sep,1)[0]):
```

Comment:
Looks like we make sure that the parent dir exists and create it if needed here.
Why don't use, for example, `pathlib`, and  `Path('.....').resolve().parent`.